impala/util.py

```diff
@@ -64,6 +64,18 @@ def as_pandas(cursor, coerce_float=False):
                                   coerce_float=coerce_float)
 
 
+def as_dict(cursor):
```

Comment:
Can you add a test for this method. E.g. see impala/tests/test_hive_dict_cursor.py